modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevel.testcase

```diff
@@ -60,4 +64,87 @@ definition {
 		}
 	}
 
+	@description = "LPS-159844. When a walkthrough step is defined and a step is below the fold, user can see popover is."
+	@priority = "3"
+	test CanRenderWhenStepIsBelowFold {
+		property portal.acceptance = "true";
+
+		task ("Given walkthrough enabled site-level") {
+			Walkthrough.addWalkthrough(fileName = "walkthrough_configuration_multiple_page.json");
+		}
+
+		task ("Given a page width larger than the preview window.") {
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 2");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 2",
+				widgetName = "Documents and Media");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 2",
+				widgetName = "JS Walkthrough Sample");
+		}
+
+		task ("When enable popover mode") {
+			Navigator.openSitePage(
+				pageName = "Walkthrough Page 1",
+				siteName = "Site Name");
+
+			Walkthrough.enablePopoverMode();
+		}
+
+		task ("And When navigate to step that is below the fold") {
+			Walkthrough.goToNextStep(key_step = "2");
+		}
+
+		task ("Then popover element is visible when rendered") {
+			AssertElementPresent(
```

Comment:
We're trying to assert that the element is visible, not present. AssertElementPresent can pass if the element is outside the viewport. I think you want AssertVisible, it will check if an element is within the viewport of browser window.